v2/component_analyses.yaml

```diff
@@ -0,0 +1,288 @@
+openapi: 3.0.1
+info:
+  title: Analytics Core API Specs for Version 2
+  version: "2"
+servers:
+- url: https://f8a-analytics-2445582058137.production.gw.apicast.io/api/v2
+  description: "Fabric8 Server"
+security:
+- tokenAuth: []
+paths:
+  /component-analyses/{eco}/{name}/{ver}:
+    get:
+      tags:
+      - Scan Services
+      summary: Get security and other details about a software component and recommendations
+        to change to a more secured version
+      operationId: f8a_scanner.api_v2.component-analyses
```

Comment:
f8a would need to be replaced. What does this field refer to ?

Reply:
operationId is an optional unique string used to identify an operation. If provided, these IDs must be unique among all operations described in your API.

 Some code generators use this value to name the corresponding methods in code.
    Links can refer to the linked operations by operationId.

Source : https://swagger.io/docs/specification/paths-and-operations/

Reply:
I still see reference to f8a

Reply:
Ack!